service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!--@1.Finding all Person records for all females-->
+    <service verb="get" noun="FemalePersons">
+        <out-parameters>
+            <parameter name="personList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="gender" value="F"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @2.Listing  the Person details with MarsMarried marital status.-->
+
+    <service verb="get" noun="Persons">
+        <out-parameters>
+            <parameter name="personList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="maritalStatusEnumId" value="MarsMarried"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @3.Finding all Order Header records so that the latest placed order appears at the top-->
+
+    <service verb="get" noun="LatestOrders">
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name=" -placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- 4.Finding the Order Header details for a given orderId.-->
+
+    <service verb="get" noun="Order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
+            <econdition field-name="orderId" value="100152"/>
+        </actions>
+    </service>
+
+    <!-- 5.Finding all Order Header records with the grandTotal greater than 50.-->
+
+    <service verb="get" noun="OrderHeaders">
+        <out-parameters>
+            <parameter name="orderHeaderDetails" type="list"/>
+        </out-parameters>
+        <actions>
+            <set field="orderHeaderDetails" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
```

Comment:
Can we put the condition in the entity find here as well, instead of iterating the list.
I think we can get all the details here instead of just a few fields.